machines/vic/certificate_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	keyFile  = "./key.pem"
+	certFile = "./cert.pem"
+)
+
+func TestGenerate(t *testing.T) {
+	log.SetLevel(log.InfoLevel)
+	if _, err := os.Stat(keyFile); err == nil {
```

Comment:
This test is to verify the certificate generation, so I cannot use a temporary file for test. But I'll use this TempFile for image upload test.